From 4fa5180387b3b89cd3c7ff66a42d06be50963eb0 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Thu, 16 Mar 2017 09:54:41 +0800
Subject: [PATCH] arm64: dts: rk3368: set higher voltage for gpu dvfs

As GPU share voltage domain with DDR and the minimum voltage of DDR
696MHz is 1100mV, GPU's voltage must be equal or greater than 1100mV.

After add ddr frequency scaling support, we can change them again.

Change-Id: I761931675265aac75425bf1cc9c7280a33f91e16
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368.dtsi b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
index 062b02fef10e..6ba70c28a8e8 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
@@ -1153,15 +1153,15 @@
 
 		opp@200000000 {
 			opp-hz = /bits/ 64 <200000000>;
-			opp-microvolt = <950000>;
+			opp-microvolt = <1100000>;
 		};
 		opp@288000000 {
 			opp-hz = /bits/ 64 <288000000>;
-			opp-microvolt = <1025000>;
+			opp-microvolt = <1100000>;
 		};
 		opp@400000000 {
 			opp-hz = /bits/ 64 <400000000>;
-			opp-microvolt = <1050000>;
+			opp-microvolt = <1100000>;
 		};
 		opp@576000000 {
 			opp-hz = /bits/ 64 <576000000>;
-- 
2.35.3

